Dockerfile

```diff
@@ -28,6 +28,9 @@ ARG CRT_FILES=""
 # Set this to the ScanCode version to use.
 ARG SCANCODE_VERSION="30.1.0"
 
+# Set this to the Python Inspector version to use.
```

Comment:
Commit message: Please add the details from the PR description to the commit message as well, to make sure it is recorded in the Git log. Also, please prefix the title with "Pip: Replace ...".

Reply:
Done

Reply:
Please also remove the ticket ids from the commit message title, it is enough to mention them in the body.